### PR TITLE
feat: youtube example app can run on large videos

### PR DESCRIPTION
**Pull request by yulaicui**
YouTube QA app is now able to run on large videos. This is enabled by separating large transcripts into chunks.

### CODE REVIEW

**Comment by jarulraj** [comment]
@yulaicui We might want to update the built-in ChatGPT UDF as opposed to adding it in this folder and importing these internal packages:

```
from eva.catalog.catalog_type import NdArrayType
from eva.configuration.configuration_manager import ConfigurationManager
from eva.udfs.abstract.abstract_udf import AbstractUDF
from eva.udfs.decorators.decorators import forward, setup
from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe
```

**Comment by yulaicui** [comment]
> @yulaicui We might want to update the built-in ChatGPT UDF as opposed to adding it in this folder and importing these internal packages:
> 
> ```
> from eva.catalog.catalog_type import NdArrayType
> from eva.configuration.configuration_manager import ConfigurationManager
> from eva.udfs.abstract.abstract_udf import AbstractUDF
> from eva.udfs.decorators.decorators import forward, setup
> from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe
> ```

I think we still need to copy the chatGPT udf because when we migrate this example to a separate repo, the user has no access to the udf file

**Comment by gaurav274** [comment]
> > @yulaicui We might want to update the built-in ChatGPT UDF as opposed to adding it in this folder and importing these internal packages:
> > ```
> > from eva.catalog.catalog_type import NdArrayType
> > from eva.configuration.configuration_manager import ConfigurationManager
> > from eva.udfs.abstract.abstract_udf import AbstractUDF
> > from eva.udfs.decorators.decorators import forward, setup
> > from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe
> > ```
> 
> @jarulraj I think we still need to copy the chatGPT udf because when we migrate this example to a separate repo, the user has no access to the udf file

We should make ChatGPT UDF inbuilt.

File: examples/youtube_qa/youtube_qa.py [hunk 1]
```diff
@@ -14,72 +14,150 @@
 # limitations under the License.
 import os
 import shutil
+import time
+
+import pandas as pd
+from pytube import YouTube, extract
+from youtube_transcript_api import YouTubeTranscriptApi
+
+from eva.interfaces.relational.db import EVADBCursor, connect
```
Comment (gaurav274) [inline examples/youtube_qa/youtube_qa.py:23]: We can now do the following.

```
import evadb
cursor = evadb.connect().cursor()
```

**Comment by yulaicui** [comment]
> > > @yulaicui We might want to update the built-in ChatGPT UDF as opposed to adding it in this folder and importing these internal packages:
> > > ```
> > > from eva.catalog.catalog_type import NdArrayType
> > > from eva.configuration.configuration_manager import ConfigurationManager
> > > from eva.udfs.abstract.abstract_udf import AbstractUDF
> > > from eva.udfs.decorators.decorators import forward, setup
> > > from eva.udfs.decorators.io_descriptors.data_types import PandasDataframe
> > > ```
> > 
> > 
> > @jarulraj I think we still need to copy the chatGPT udf because when we migrate this example to a separate repo, the user has no access to the udf file
> 
> We should make ChatGPT UDF inbuilt. I agree with @jarulraj; we should not scare the users with these imports.

makes sense. I'm creating a separate PR to address this ASAP. The PR will add chatgpt as builtin udf.

**Pull request closed without merging**